UPGRADE-14.0.md

```diff
@@ -544,6 +544,35 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   check the visibility of properties ([#2944](https://github.com/shopsys/shopsys/pull/2944))
     -   `Shopsys\FrameworkBundle\Form\Transformers\WysiwygCdnDataTransformer::$cdnFacade` is now protected
     -   `Shopsys\FrameworkBundle\Model\ImageSitemap\ImageSitemapCronModule::$imageSitemapFacade` is now protected
+-   improve memory usage of feed exports and crons
+    -   `Shopsys\FrameworkBundle\Model\Feed\FeedExport::__construct()` changed its interface:
+    ```diff
+        public function __construct(
+            protected readonly FeedInterface $feed,
+            protected readonly DomainConfig $domainConfig,
+            protected readonly FeedRenderer $feedRenderer,
+            protected readonly FilesystemOperator $filesystem,
+            protected readonly Filesystem $localFilesystem,
+            protected readonly MountManager $mountManager,
+            protected readonly EntityManagerInterface $em,
+            protected readonly string $feedFilepath,
+            protected readonly string $feedLocalFilepath,
+    +       protected readonly ServicesResetter $servicesResetter,
+            protected ?int $lastSeekId = null,
+        )
+    ```
+    -   `Shopsys\FrameworkBundle\Model\Feed\FeedExportFactory::__construct()` changed its interface:
+    ```diff
+        public function __construct(
+            protected readonly FeedRendererFactory $feedRendererFactory,
+            protected readonly FilesystemOperator $filesystem,
+            protected readonly EntityManagerInterface $em,
+            protected readonly FeedPathProvider $feedPathProvider,
+            protected readonly Filesystem $localFilesystem,
+            protected readonly MountManager $mountManager,
+    +       protected readonly ServicesResetter $servicesResetter,
+        )
+    ```
```

Comment:
isn't the #see project-base diff missing?

Reply:
added